project/bin/src/main/webapp/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
do we need this file? seems like a copy from starter project? 
(same for script.js below)

Reply:
i think alex updated index.html in the create-home-page branch so it's updated I'll check for script.js right now

Reply:
you might just have to re pull from master to sync with the latest version